properties/dapr_managed_netcore.props

```diff
@@ -5,6 +5,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <WarningLevel>4</WarningLevel>
+    <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
```

Comment:
This is needed when you run `dotnet pack` on a solution - otherwise you get a bunch of warnings about test projects and samples which aren't packable.